pushToTamrNexus.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
There's a way, apparently, to get Gradle to do this. Something to clean up after Chester, I think.